src/sentry/static/sentry/app/components/stream/group.tsx

```diff
@@ -251,14 +262,14 @@ const StreamGroup = createReactClass({
           mr={1}
           flex="1"
         >
-          <EventOrGroupHeader data={data} query={query} />
+          <EventOrGroupHeader includeLink data={data} query={query} size="normal" />
           <EventOrGroupExtraDetails data={data} />
         </GroupSummary>
         {hasGuideAnchor && <GuideAnchor target="issue_stream" />}
         {withChart && (
           <Box width={160} mx={2} className="hidden-xs hidden-sm">
             <GroupChart
-              statsPeriod={statsPeriod}
+              statsPeriod={statsPeriod!}
```

Comment:
Are you sure about it? I am always trying to avoid the usage of `something!`

Reply:
statsPeriod has a default prop, but is currently somewhat broken because of the higher order functions on export. Had to cast default props to partial...